### DISCUSSION TITLE
In version 7.3.3, using `![]()` to insert images will be automatically escaped to `!\[]\()`

### DISCUSSION THREAD

**Discussion by KunMoe** [Milkdown/milkdown#1219] [Q&A] [answered]
After updating Milkdown to version 7.3.3, using the syntax `![]()` to insert images leads to an inability to insert images. 
According to the output, this issue seems to occur because Milkdown is escaping `![]()` to `!\[]\()`. Is this the expected behavior? 
![](https://cdn.jsdelivr.net/gh/kun-moe/kun-image@main/blog/202401080011919.png)
This issue does not occur in version 7.3.2. 
I've created [an example](https://github.com/KUN1007/milkdown-vue3-demo) to reproduce the issue.
Any clarification on this matter would be greatly appreciated.

**Comment by reonokiy** [comment, marked as answer]
check this issue https://github.com/Milkdown/milkdown/issues/1278

**Reply by KunMoe** [reply]
Thanks! It works fine!
